src/components/sdk/MeetingControls/docs/VideoInputBackgroundBlurControl.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import VideoInputBackgroundBlurControl from '../VideoInputBackgroundBlurControl';
+
+<Meta title="SDK Components/MeetingControls/VideoInputBackgroundBlurControl" />
+
+# VideoInputBackgroundBlurControl
+
+The `VideoInputBackgroundBlurControl` component renders a `ControlBarButton` with pop over menu options to select through multiple video input sources.
+These video input sources are provided by the `useVideoInputs` hook.
+The `VideoInputBackgroundBlurControl` component also has a checkbox that will enable or disable Background Blur on the chosen video input source.
+If you want to apply Background Blur to your video input device stream, try this `VideoInputBackgroundBlurControl`. If not, `VideoInputControl` is a better choice.
```

Comment:
Should we just say "background blur" as the JS SDK PR says so but not captalized?